File: R/ex1004.r
#input <- file('stdin', 'r')

#NumA <- as.integer(readline(input, n = 1))
#NumB <- as.integer(readline(input, n = 1))

#NumA = NumA + NumB

#write(paste0("X =", NumA, ''))
input <- file("stdin", "r")

num_a <- as.integer(readLines(input, n = 1))
num_b <- as.integer(readLines(input, n = 1))
produto <- num_a * num_b
cat("PROD = ", produto, "\n", sep = "")

File: Java/ex1004.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
	public static void main (String[] args) throws IOException {
		InputStreamReader ir = new InputStreamReader(System.in);
		BufferedReader in = new BufferedReader(ir);

		int a, b, prod;

		a = Integer.parseInt(in.readLine());
		b = Integer.parseInt(in.readLine());

		prod = a * b;

		System.out.printf("PROD = %d\n", prod);
	}
}
